src/Models/EntityArchive.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace KusikusicmsModels\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class EntityArchive extends Model
+{
+
```

Comment:
Maybe adding some constants like if its a backup or a draft

Reply:
Also, would be a good idea if we are using this for drafts, that we can relate a record to a user